workflows/quickstart/main.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Simple CLI to run the executeWorkflow function.
+// Used for one-off testing and development.
+
+// [START workflows_api_quickstart]
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	executions "cloud.google.com/go/workflows/executions/apiv1beta"
+	executionspb "google.golang.org/genproto/googleapis/cloud/workflows/executions/v1beta"
+)
+
+// executeWorkflow executes a workflow and returns the results from the workflow.
+func executeWorkflow(projectID, locationID, workflowID string) (string, error) {
+	ctx := context.Background()
+
+	// Creates a client.
+	client, err := executions.NewClient(ctx)
+	if err != nil {
+		return "", err
+	}
+
+	if workflowID == "" {
+		workflowID = "myFirstWorkflow"
+	}
+	workflowPath := fmt.Sprintf("projects/%s/locations/%s/workflows/%s", projectID, locationID, workflowID)
+
+	exe, err := client.CreateExecution(ctx, &executionspb.CreateExecutionRequest{
+		Parent: workflowPath,
+	})
+	if err != nil {
+		return "", err
+	}
+
+	name := exe.GetName()
+	fmt.Fprintf(os.Stdout, "Created execution: %v\n", name)
+
+	// Wait for execution to finish, then print results.
+	backoffDelay := 1 * time.Second // Start wait with delay of 1s.
+	fmt.Println("Poll for result...")
+	for {
+		e, err := client.GetExecution(ctx, &executionspb.GetExecutionRequest{
```

Comment:
nit: The variables in this sample include exe, e, and err. That's a bit confusing to follow. Maybe this should overwrite the initial return of the execution since we don't need it anymore?

```suggestion
		exe, err = client.GetExecution(ctx, &executionspb.GetExecutionRequest{
```